packages/store-ui/src/atoms/Radio/stories/Radio.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import type { Story, Meta } from '@storybook/react'
+import React from 'react'
+
+import type { ComponentArgTypes } from '../../../typings/utils'
+import type { RadioProps } from '../Radio'
+import Component from '../Radio'
+import mdx from './Radio.mdx'
+
+const RadioTemplate: Story<RadioProps> = (props) => <Component {...props} />
+
+export const Radio = RadioTemplate.bind({})
+
+const controls: ComponentArgTypes<RadioProps> = {
+  checked: {
+    control: {
+      type: 'boolean',
+    },
+  },
+}
+
+const actions: ComponentArgTypes<RadioProps> = {
+  onClick: { action: 'clicked', table: { disable: true } },
+}
+
+export default {
+  title: 'Atoms/Radio',
+  component: Radio,
+  argTypes: {
```

Comment:
Just to standardize the code, I think it's better to make a `const argTypes = { controls: ..., actions: ... }`. We're doing this way in the other components too.

Reply:
I do not think so, because we would have a const to `parameters` too